langchain/src/prompts/chat.ts

```diff
@@ -158,9 +158,33 @@ export class MessagesPlaceholder<
     return [this.variableName];
   }
 
+  validateInputOrThrow(input: Array<unknown>): input is BaseMessage[] {
+    let isInputBaseMessage = false;
+
+    if (Array.isArray(input)) {
+      isInputBaseMessage = input.every((message) =>
+        isBaseMessage(message as BaseMessage)
+      );
+    } else {
+      isInputBaseMessage = isBaseMessage(input as BaseMessage);
```

Comment:
Can you change `isBaseMessage` to tolerate `null/undefined` values?

https://github.com/langchain-ai/langchainjs/blob/main/langchain/src/schema/index.ts#L461